engine/src/gldrv/gl_light.cpp

```diff
@@ -178,6 +178,31 @@ void /*GFXDRVAPI*/ GFXLight::SetProperties(enum LIGHT_TARGET lighttarg, const GF
     apply_attenuate(attenuated());
 }
 
+void /*GFXDRVAPI*/ GFXLight::SetProperties(const enum LIGHT_TARGET light_target, const Vector& vector) {
```

Comment:
Where did this code come from?
I think you took my comments in the wrong direction. The old devs were using `Vector` as a convenience struct for storing `GFXColor` (without alpha).
I wanted to get rid of the convenience as it was wrong. That's it.

I noted one place where it was clear they were doing more. There we should locally extract the tuple (i,j,k) and create GFXColor. But, looking deeper, I'm not even sure I understand what's going on at all.
`Called wrong overload for this property, with Vector& instead of GFXColor&`
but the parameter is `Vector`?!

Maybe you should write a brief annotation of where you're going here because I got lost.

Reply:
See note above.

Reply:
I mean, it's possible.